polynote-kernel/src/main/scala/polynote/kernel/lang/scal/package.scala

```diff
@@ -7,6 +7,4 @@ import scala.reflect.api.Universe
   */
 package object scal {
```

Comment:
can we remove the package object if it no longer holds anything?

Reply:
I'd like to keep it there if only just for the comment explaining why it's `scal`.

Reply:
Fair enough 😃